Source/WebCore/PAL/pal/spi/cocoa/PassKitSPI.h

```diff
@@ -71,6 +71,12 @@
 #endif
 #endif
 
+#if HAVE(PASSKIT_PAYMENT_ORDER_DETAILS) && USE(APPLE_INTERNAL_SDK)
+#import <PassKitCore/PKPaymentRequestStatus.h>
+#else
+#import <PassKit/PKPaymentRequestStatus.h>
+#endif
```

Comment:
This says that we import `PassKitCore/PKPaymentRequestStatus.h` from macOS 13 internal SDK, and `PassKit/PKPaymentRequestStatus.h` in all other cases. And it's based on target OS, not on SDK version.

This feels very unusual, is this indeed the intention?

Reply:
Not intentional. Looks like I should follow the above convention, and use:
```
#if HAVE(PASSKIT_PAYMENT_ORDER_DETAILS)
#if HAVE(PASSKIT_MODULARIZATION) && USE(APPLE_INTERNAL_SDK)
```